dist/templates/ovn-ipsec.yaml.j2

```diff
@@ -230,6 +230,12 @@ spec:
           # Workaround for https://github.com/libreswan/libreswan/issues/373
           ulimit -n 1024
 
+          # Remove stale pluto.pid if no pluto process is running
```

Comment:
who writes this pid file? does it mean pluto exited early and didnt remove its pid file or ?

Reply:
ye, this creates pluto.pid file https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/dist/templates/ovn-ipsec.yaml.j2#L239 and it gets killed upon sigterm signal https://github.com/huiran0826/ovn-kubernetes/blob/ipsec-metric-e2e/dist/templates/ovn-ipsec.yaml.j2#L203.